packages/default-theme/components/account/MyAddresses.vue

```diff
@@ -0,0 +1,86 @@
+<template>
```

Comment:
There is no possibility to add a new address.

Reply:
yes, it's true. It wasn't planned to handle the forms. We need your powers here ;)